plugin/metrics/prometheus/options.go

```diff
@@ -33,7 +33,7 @@ const (
 	defaultTokenFilePath  = ""
 
 	defaultSupportSpanmetricsConnector = true
-	defaultMetricNamespace             = ""
+	defaultMetricNamespace             = "traces_span_metrics"
```

Comment:
@yurishkuro setting this default here seems to take care of the v2 test but the v1 test is still getting an empty response. do you have any idea why that could be? both v1 and v2 read this config.

Reply:
this was actually not related to the changes in this PR and was failing due to missing http receiver endpoint in the otel collector config